### ISSUE TITLE
Extend API loadLocationData endpoints for LoD2 data in getting volume

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#163]
## Description
See Epic: #160 

## Depends on
#161 

## User story
* Get a volume

## Acceptance criteria
* FE request data via API endpoint
* BE send data to frontend

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
